teku/src/main/java/tech/pegasys/teku/cli/deposit/DepositTransactionSender.java

```diff
@@ -128,7 +130,7 @@ public BigInteger getGasLimit(final String contractFunc) {
 
     @Override
     public BigInteger getGasLimit() {
-      return BigInteger.valueOf(200_000L);
+      return BigInteger.valueOf(REASONABLE_GAS_LIMIT);
```

Comment:
This constant is redundant and just makes the code harder to read.  We're literally in a method called `getGasLimit` so I think it's clear what the `200_000` means and `REASONABLE_GAS_LIMIT` doesn't provide any additional information, but does make the actual value one step further removed so harder to find.

Reply:
agree, if we were going to do that you'd be better off just having a constant declared and referencing the constant rather than calling another function to access it...